puppet/manifests/default.pp

```diff
@@ -25,19 +25,8 @@
   $controller_ip = '192.168.0.2'
   $worker_ip = '192.168.0.3'
 
-  class { 'hosts':
-    enable_fqdn_entry => false,
-    host_entries      => {
-      'controller.seed-stack.local' => {
-        ip           => $controller_ip,
-        host_aliases => ['controller']
-      },
-      'worker.seed-stack.local' => {
-        ip           => $worker_ip,
-        host_aliases => ['worker']
-      }
-    }
-  }
+  host { 'controller.seed-stack.local': ip => $controller_ip }
```

Comment:
Can you also do `host_aliases` to add aliases to the hostnams please? It's actually needed for some things to work because we're currently using hostnames and not FQDNs for mesos/marathon hostnames.